container-chains/templates/frontier/runtime/src/xcm_config.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+use {
+    super::{
+        AccountId, AllPalletsWithSystem, Balances, ParachainInfo, ParachainSystem, PolkadotXcm,
+        Runtime, RuntimeCall, RuntimeEvent, RuntimeOrigin, WeightToFee, XcmpQueue,
+    },
+    ccp_xcm::SignedToAccountKey20,
+    frame_support::{
+        parameter_types,
+        traits::{Everything, Nothing, PalletInfoAccess},
+        weights::Weight,
+    },
+    frame_system::EnsureRoot,
+    pallet_xcm::XcmPassthrough,
+    sp_core::ConstU32,
+    xcm::latest::prelude::*,
+    xcm_builder::{
+        AccountKey20Aliases, AllowKnownQueryResponses, AllowSubscriptionsFrom,
+        AllowTopLevelPaidExecutionFrom, CurrencyAdapter, EnsureXcmOrigin, FixedWeightBounds,
+        IsConcrete, ParentIsPreset, RelayChainAsNative, SiblingParachainAsNative,
+        SiblingParachainConvertsVia, SignedAccountKey20AsNative, SovereignSignedViaLocation,
+        TakeWeightCredit, UsingComponents, WithComputedOrigin,
+    },
+    xcm_executor::XcmExecutor,
+};
+
+parameter_types! {
+    // Self Reserve location, defines the multilocation identifiying the self-reserve currency
+    // This is used to match it also against our Balances pallet when we receive such
+    // a MultiLocation: (Self Balances pallet index)
+    // We use the RELATIVE multilocation
+    pub SelfReserve: MultiLocation = MultiLocation {
+        parents:0,
+        interior: Junctions::X1(
+            PalletInstance(<Balances as PalletInfoAccess>::index() as u8)
+        )
+    };
+
+    // One XCM operation is 1_000_000_000 weight - almost certainly a conservative estimate.
+    pub UnitWeightCost: Weight = Weight::from_parts(1_000_000_000, 64 * 1024);
+
+    // TODO: revisit
+    pub const RelayNetwork: NetworkId = NetworkId::Polkadot;
+
+    // The relay chain Origin type
+    pub RelayChainOrigin: RuntimeOrigin = cumulus_pallet_xcm::Origin::Relay.into();
+
+    pub const MaxAssetsIntoHolding: u32 = 64;
```

Comment:
Shouldn't be better to use instead the `MAX_ASSETS` constant that comes from xcm_primitives? Just in case this changes in the future and so that we can have a better track of it.
```suggestion
    pub const MaxAssetsIntoHolding: u32 = xcm_primitives::MAX_ASSETS;
```